TheFourFifths/consus
# Hotfix/student id to number

### Summary

StudentId's were not consistently being used as numbers, and there was an enforcement to have them be strings, changed the enforcement to be for numbers(because that's what ids are) and the tests associated with that.

### Checklist


### How to Review

1. npm test?
2. Run client on DEV and server on this branch
3. Lookup student id 123456
4. Try to checkout an item
5. Verify checkout works
6. Checkin an item
7. Verify checkin works

### Links
